lib/pay/billable.rb

```diff
@@ -83,6 +85,12 @@ def upcoming_invoice
 
     private
 
+    def update_processor_email
+      if saved_change_to_email? && subscriptions.any?
```

Comment:
I think we can swap the `subscriptions.any?` with `processor_id?` that way it updates them even if they don't have a subscription, but may have previously.